webapp/tests/karma/unit/services/db-sync.js

```diff
@@ -66,13 +70,13 @@ describe('DBSync service', () => {
       { id: 'c' }
     ] }));
     return service().then(() => {
-      chai.expect(allDocs.callCount).to.equal(1); // 1 'from' calls, 0 'to' calls
+      chai.expect(allDocs.callCount).to.equal(0);
       chai.expect(Auth.callCount).to.equal(1);
       chai.expect(Auth.args[0][0]).to.equal('can_edit');
       chai.expect(from.callCount).to.equal(1);
       chai.expect(from.args[0][1].live).to.equal(true);
       chai.expect(from.args[0][1].retry).to.equal(true);
-      chai.expect(from.args[0][1].doc_ids).to.deep.equal(['m','e','d','i','c']);
+      chai.expect(from.args[0][1].doc_ids).to.deep.equal(undefined);
```

Comment:
Can we deep equal the whole `from.args[0][1]` which will prove the doc_ids is null without having to explicitly list it?